bin/ch-test

```diff
@@ -840,10 +839,22 @@ if [[ $phase = one-file ]]; then
     fi
 fi
 
+# Kludge for autoconf make check integration.
+if [[ -z $phase && -n $CH_TEST_PHASE ]]; then
+    case $CH_TEST_PHASE in
+    all|build|examples|run)
+        phase=$CH_TEST_PHASE
+    ;;
+    *)
+    ;;
+    esac
+fi
+
```

Comment:
I think we should just copy `$CH_TEST_PHASE` to `$phase` without checking it. That will be sufficient for internal purposes, and at some point in the future we can write code that checks for valid phase just once.